docs/source/release.rst

```diff
@@ -7,6 +7,7 @@ Release Notes
    These release notes are for versions of ibis **1.0 and later**. Release
    notes for pre-1.0 versions of ibis can be found at :doc:`/release-pre-1.0`
 
+* :feature:`1997` Add initial sql server backend support
```

Comment:
@costrouc could you move your release note just below `* :release:`1.2.1 <pending>`` ?

Reply:
Moved. Thanks! Didn't want to ask you to include it in the 1.2.1 release but I appreciate that it will be